.travis.yml

```diff
@@ -28,6 +28,7 @@ cache:
 before_install:
```

Comment:
I'm not certain, but we have it set to 3G at the moment and I haven't seen any memory-related errors